application/backend/src/main/java/io/typerefinery/websight/models/components/Smallticker.java

```diff
@@ -0,0 +1,52 @@
+package io.typerefinery.websight.models.components;
+
+import static org.apache.sling.models.annotations.DefaultInjectionStrategy.OPTIONAL;
+
+import javax.inject.Inject;
+import lombok.Getter;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.apache.sling.models.annotations.Default;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.injectorspecific.SlingObject;
+
+@Model(adaptables = Resource.class, defaultInjectionStrategy = OPTIONAL)
+public class Smallticker {
```

Comment:
Please follow proper naming convention 

Smallticker to SmallTicker

Reply:
Done